lbry/extras/cli.py

```diff
@@ -262,6 +262,8 @@ def setup_logging(logger: logging.Logger, args: argparse.Namespace, conf: Config
 
 
 def run_daemon(args: argparse.Namespace, conf: Config):
+    if conf.needs_proactor:
+        asyncio.set_event_loop(asyncio.ProactorEventLoop())
```

Comment:
This can just be (instead of adding a non-config related method to `Config`):
```
    if sys.version_info < (3, 8) and platform.system() == "Windows":
        # TODO: remove after we move to requiring Python 3.8
        asyncio.set_event_loop(asyncio.ProactorEventLoop())
```

Reply:
Fixed and pushed.